src/plugins/robots/drone/simulator/pointmass3d_drone_model.h

```diff
@@ -125,6 +130,28 @@ namespace argos {
       const static Real YAW_KP; 
       const static Real YAW_KI; 
       const static Real YAW_KD; 
+      /* sensors noise constants*/
+      const static Real MEMS_GYRO_NOISE_MEAN;
+      const static Real MEMS_GYRO_BIAS_MEAN;
+      const static Real MEMS_GYRO_BIAS_STD_DEV;
+      const static Real MEMS_GYRO_NOISE_STD_DEV_X;
+      const static Real MEMS_GYRO_NOISE_STD_DEV_Y;
+      const static Real MEMS_GYRO_NOISE_STD_DEV_Z;
+      const static Real MEMS_GYRO_RANDOM_WALK_INIT;
+      const static Real MEMS_GYRO_BIAS_INIT;
+      const static Real MEMS_ACCEL_NOISE_MEAN;
+      const static Real MEMS_ACCEL_BIAS_MEAN;
+      const static Real MEMS_ACCEL_BIAS_STD_DEV;
+      const static Real MEMS_ACCEL_NOISE_STD_DEV_X;
+      const static Real MEMS_ACCEL_NOISE_STD_DEV_Y;
+      const static Real MEMS_ACCEL_NOISE_STD_DEV_Z;
+      const static Real MEMS_ACCEL_RANDOM_WALK_INIT;
+      const static Real MEMS_ACCEL_BIAS_INIT;
+      
+      protected:
```

Comment:
I don't think this should be `protected`. Please move this variable above the constants and remove the access specifier.